### PR TITLE
Additional tickets to be added to RN for 2.41.2

### PR DESCRIPTION
**Pull request by rasika-chivate**
Additional fixed issues to be added to the Release Notes for PMM 2.41.2.

### CODE REVIEW

**Comment by rasika-chivate** [comment]
@BupycHuk Please provide the correct the description for https://perconadev.atlassian.net/browse/PMM-12986 and https://perconadev.atlassian.net/browse/PMM-12997 as I am not sure of the descriptions in the RN.

File: docs/release-notes/2.41.2.md [hunk 1]
```diff
@@ -89,26 +89,41 @@ Read our [blog post](https://www.percona.com/blog/monitoring-a-postgresql-patron
...
+
+- [PMM-12785](https://perconadev.atlassian.net/browse/PMM-12785) - Fixed an issue where VictoriaMetrics scrape configuration contained unsupported parameters. 
+
+- [PMM-12805](https://perconadev.atlassian.net/browse/PMM-12805) - While monitoring MongoDB servers, the disk space was being consumed due to the `CommandNotSupportedOnView` message in the logs. This issue has now been resolved.
+
+- [PMM-12809](https://perconadev.atlassian.net/browse/PMM-12809) - Fixed several CVEs in PMM versions v2.40.1 and above.
+
+- [PMM-12852](https://perconadev.atlassian.net/browse/PMM-12852) - An internal server error was thrown while attempting to modify any setting in PMM. This issue has been resolved now.
+
+- [PMM-12986](https://perconadev.atlassian.net/browse/PMM-12986) - Fixed an issue related to the `PostgreSQL connections in use` template which was previously not functioning for Alerts.
```
Comment (BupycHuk) [inline docs/release-notes/2.41.2.md:120]: it was firing false positively

File: docs/release-notes/2.41.2.md [hunk 2]
```diff
@@ -89,26 +89,41 @@ Read our [blog post](https://www.percona.com/blog/monitoring-a-postgresql-patron
...
 
-[PMM-12348](https://perconadev.atlassian.net/browse/PMM-12348) - The ClickHouse engine was updated from the ordinary engine to the Atomic engine, which is the most robust engine, after upgrading PMM to version 2.41.0.
+- [PMM-12894](https://perconadev.atlassian.net/browse/PMM-12894) - We have limited the number of connections used by postgresql_exporter. Additionally, we have fixed an issue where connections are closed as soon as they are no longer needed, thereby preventing any hanging or idle connections.
```
Comment (BupycHuk) [inline docs/release-notes/2.41.2.md:122]: it's an improvement that should improve the way PMM affects user environment(less CPU usage). And it's postgres_exporter.

File: docs/release-notes/2.41.2.md [hunk 3]
```diff
@@ -89,26 +89,41 @@ Read our [blog post](https://www.percona.com/blog/monitoring-a-postgresql-patron
...
 
-[PMM-12785](https://perconadev.atlassian.net/browse/PMM-12785) - Fixed an issue where VictoriaMetrics scrape configuration contained unsupported parameters. 
+- [PMM-12897](https://perconadev.atlassian.net/browse/PMM-12897) - Previously, two heavy queries were running for each database in the PostgreSQL server at medium resolution, which impacted the performance. These queries have now been moved to low resolution to improve performance.
```
Comment (BupycHuk) [inline docs/release-notes/2.41.2.md:124]: it's an improvement.

File: docs/release-notes/2.41.2.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/release-notes/2.41.2.md:120]: @BupycHuk Is this correct: The **PostgreSQL connections in use** alert was triggering false positive notifications, but the issue has been resolved now.

File: docs/release-notes/2.41.2.md (hunk 2 above)
Comment (rasika-chivate) [reply docs/release-notes/2.41.2.md:122]: Fixed

File: docs/release-notes/2.41.2.md (hunk 1 above)
Comment (BupycHuk) [reply docs/release-notes/2.41.2.md:121]: looks good to me

File: docs/release-notes/2.41.2.md (hunk 3 above)
Comment (rasika-chivate) [reply docs/release-notes/2.41.2.md:124]: Fixed.

**Review by BupycHuk — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**